### PR TITLE
chore(web-components): Add notes on Yarn 2 and PnP

### PR DESCRIPTION
**Pull request by muinmomin**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

Didn't include an issue or a change file since it's a quick README update from the GitHub editor. Hope this is okay.

#### Description of changes

Added some notes on how to get the Fluent web components working with Yarn 2 and PnP.

#### Focus areas to test

N/A

### CODE REVIEW

File: packages/web-components/README.md [hunk 1]
```diff
@@ -55,3 +55,22 @@ Storybook will watch modules for changes and hot-reload the module when necessar
 `Failed to execute 'define' on 'CustomElementRegistry': the name "my-custom-element-name" has already been used with this registry`
 
 This is a known issue and will indicate that you need to refresh the page. We're working on surfacing a more instructive error message for this case.
+
+## Notes
+
+### Using Yarn 2 with PnP
+
+`@microsoft/fast-components` works out of the box with Yarn 2 and PnP; however, because of how Yarn expects peer dependencies to be satisfied, we need additional configuation for `fluentui/web-components`.
```
Comment (chrisdholt) [inline packages/web-components/README.md:63]: I’m not sure we need to add a reference as this package is technically unrelated. If we can rework this to omit the reference to the fast-components everything else looks relatively straight forward.

Comment (muinmomin) [reply packages/web-components/README.md]: Good point, updated!

**Review by merceyz — COMMENTED** [review]
This is odd, why wouldn't you (anyone) just fix it in the respective packages instead?

**Comment by layershifter** [comment]
@chrisdholt can you please check this PR?

**Comment by EisenbergEffect** [comment]
@chrisdholt Any thoughts on this?

**Review by chrisdholt — APPROVED** [review]
